pennylane/_device.py

```diff
@@ -749,6 +749,14 @@ def batch_transform(self, circuit: QuantumTape):
         # device property present in braket plugin
         use_grouping = getattr(self, "use_grouping", True)
 
+        if any(isinstance(m, MidMeasureMP) for m in circuit.operations):
```

Comment:
Is it related to this PR? If yes why is it needed?

Reply:
I think it's a merge artifact. I'll fix it.